omnigibson/robots/manipulation_robot.py

```diff
@@ -2,6 +2,7 @@
 from collections import namedtuple
 import numpy as np
 import networkx as nx
+from telemoma.human_interface.teleop_core import TeleopAction
```

Comment:
Don't import this here. Creates an unnecessary dependency. Remove the type hint it's used for.